plugins/kinetic-dp/fu-kinetic-dp-aux-dpcd.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2021 Jeffrey Lin <jlin@kinet-ic.com>
+ * Copyright (C) 2022 Hai Su <hsu@kinet-ic.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "fu-kinetic-dp-aux-dpcd.h"
+#include <fwupdplugin.h>
+
+gboolean
+fu_kinetic_dp_aux_dpcd_read_oui(FuKineticDpConnection *connection,
+                                                                                                    guint8 *buf,
+                                                                                                    guint32 buf_size,
+                                                                                                    GError **error)
+{
+  g_debug("aux dpcd read OUI...");
+  if (buf_size < DPCD_SIZE_IEEE_OUI) {
+    g_set_error(error,FWUPD_ERROR,FWUPD_ERROR_INTERNAL,
+                              "aux dpcd read buffer size [%u] is too small to read IEEE OUI",buf_size);
+    return FALSE;
+  }
+  if (!fu_kinetic_dp_connection_read(connection,DPCD_ADDR_IEEE_OUI,
+                                                                         buf,DPCD_SIZE_IEEE_OUI,error)){
+    g_prefix_error(error, "aux dpcd read OUI failed: ");
+    return FALSE;
+  }
+  return TRUE;
+}
+
+gboolean
+fu_kinetic_dp_aux_dpcd_write_oui(FuKineticDpConnection *connection,
+                                                                                                     const guint8 *buf,
+                                                                                                     GError **error)
+{
+  g_debug("aux dpcd write OUI...");
+  if (!fu_kinetic_dp_connection_write(connection,  DPCD_ADDR_IEEE_OUI,
+                                                                          buf, DPCD_SIZE_IEEE_OUI, error)) {
+    g_prefix_error(error, "aux dpcd write OUI failed: ");
+    return FALSE;
+  }
+  return TRUE;
+}
+
+gboolean
+fu_kinetic_dp_aux_dpcd_read_branch_id_str(
+                                                                                                    FuKineticDpConnection *connection,
+                                                                                                    guint8 *buf,
+                                                                                                    guint32 buf_size,
+                                                                                                    GError **error)
```

Comment:
all the whitespace is terrible; can you run ./contrib/setup and make sure that the clang auto-formatter is set up properly please.

Reply:
I reformat with clang-format after running contrib/setup in my environment and push it up again.